### PR TITLE
BER-PT-Olga-Array Exercise

### PR DESCRIPTION
**Pull request by Olga-OK**

### CODE REVIEW

File: starter-code/src/functions-and-arrays.js [hunk 1]
```diff
@@ -1,18 +1,86 @@
...
+
+function calculateAvg (numbersAvg) {
+  for (let i = 0; i < numbersAvg.length; i++){
+    numbersAvg[i] += totalSum
+    totalSum = numbersAvg[i]
+  }
+  console.log (totalSum)
+}
+
+calculateAvg (numbersAvg)
```
Comment (numbing) [inline starter-code/src/functions-and-arrays.js]: its nice but also you could use sumArray function to get the average

File: starter-code/src/functions-and-arrays.js [hunk 2]
```diff
@@ -25,6 +93,15 @@ const wordsArr = [
...
 
+function averageWordLength(whatever) {
+  for (let i = 0; i < whatever.length; i++){
+    sumOfLetters += whatever[i].length    
+  }
+  console.log (sumOfLetters/whatever.length)
+}
+
+averageWordLength (words)
+
```
Comment (numbing) [inline starter-code/src/functions-and-arrays.js]: well done

File: starter-code/src/functions-and-arrays.js [hunk 3]
```diff
@@ -1,18 +1,86 @@
 // Iteration #1: Find the maximum
+function findMax (number1, number2) {
```
Comment (Piepongwong) [inline starter-code/src/functions-and-arrays.js]: TR--

File: starter-code/src/functions-and-arrays.js [hunk 4]
```diff
@@ -1,18 +1,86 @@
...
+    console.log (`Max value is ${number1}`)
+  } else {
+    console.log (`Max value is ${number2}`)
+  }
+}
+ findMax (1001,100)
+
+ // Second Option
+
+ function findMax2 (number1, number2) {
```
Comment (Piepongwong) [inline starter-code/src/functions-and-arrays.js]: CQ++

**Pull request closed without merging**